src/meili/candidate_search.cc

```diff
@@ -126,18 +128,6 @@ CandidateCollector::WithinSquaredDistance(const midgard::PointLL& location,
   return candidates;
 }
 
-std::vector<std::vector<baldr::PathLocation>>
-CandidateQuery::QueryBulk(const std::vector<midgard::PointLL>& locations,
```

Comment:
this wasnt used anywhere. it was part of a PR that @SiarheiFedartsou had done to make it possible to inject different querying mechanisms but ended up not being used.